src/Symfony/Component/Validator/Tests/Constraints/EmailValidatorTest.php

```diff
@@ -91,6 +91,7 @@ public function getInvalidEmails()
             array('example'),
             array('example@'),
             array('example@localhost'),
+            array('foo@example.com bar'),
```

Comment:
http://en.wikipedia.org/wiki/Email_address

`admin@mailserver1 (local domain name with no TLD)`

I'm not sure about the RFC, but it seems that the simple test should allow the localhost example.